src/assets/simp/build/simp.spec

```diff
@@ -135,8 +135,8 @@ Requires: pupmod-puppet-gitlab >= 3.0.2
 Requires: pupmod-puppet-posix_acl >= 0.1.1
 Requires: pupmod-puppet-snmp >= 4.1.0
 Requires: pupmod-puppetlabs-docker >= 3.5.0
-Requires: pupmod-puppetlabs-java >= 2.4.0
-Requires: pupmod-puppetlabs-mysql >= 5.3.0
+Requires: pupmod-puppetlabs-java >= 3.3.0
```

Comment:
Please revert to 2.4.0.  We don't want to force our users to upgrade a major version for a module we don't use.